### ISSUE TITLE
Source Intercom: extra slash in Intercom base url

### ISSUE THREAD

**Issue by mrhallak** [airbytehq/airbyte#11177]
## Environment
- **Airbyte version**:  0.35.55-alpha
- **OS Version / Instance**: Ubuntu 20.04
- **Deployment**: Docker compose
- **Source Connector and version**: Intercom 0.1.13
- **Severity**: Medium
- **Step where error happened**: Setup new connection

## Current Behavior
Unable to test the connection

## Expected Behavior
Being able to test the connection

## Logs
*If applicable, please upload the logs from the failing operation. 
For sync jobs, you can download the full logs from the UI by going to the sync attempt page and 
clicking the download logs button at the top right of the logs display window.*

<details>
<summary>LOG</summary> 

```
2022-03-15 19:30:35 INFO i.a.w.t.TemporalAttemptExecution(get):105 - Docker volume job log path: /tmp/workspace/849ce07d-1ad2-4664-87e1-35ed5e28a78b/0/logs.log
2022-03-15 19:30:35 INFO i.a.w.t.TemporalAttemptExecution(get):110 - Executing worker wrapper. Airbyte version: 0.35.55-alpha
2022-03-15 19:30:35 INFO i.a.c.i.LineGobbler(voidCall):82 - Checking if airbyte/source-intercom:0.1.13 exists...
2022-03-15 19:30:35 INFO i.a.w.p.DockerProcessFactory(create):106 - Creating docker job ID: 849ce07d-1ad2-4664-87e1-35ed5e28a78b
2022-03-15 19:30:35 INFO i.a.c.i.LineGobbler(voidCall):82 - airbyte/source-intercom:0.1.13 was found locally.
2022-03-15 19:30:35 INFO i.a.w.p.DockerProcessFactory(create):158 - Preparing command: docker run --rm --init -i -w /data/849ce07d-1ad2-4664-87e1-35ed5e28a78b/0 --log-driver none --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e WORKER_JOB_ATTEMPT=0 -e WORKER_CONNECTOR_IMAGE=airbyte/source-intercom:0.1.13 -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_VERSION=0.35.55-alpha -e WORKER_JOB_ID=849ce07d-1ad2-4664-87e1-35ed5e28a78b airbyte/source-intercom:0.1.13 check --config source_config.json
2022-03-15 19:30:37 WARN c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword existingJavaType - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2022-03-15 19:30:37 ERROR i.a.w.p.a.DefaultAirbyteStreamFactory(internalLog):95 - Check failed
2022-03-15 19:30:37 INFO i.a.w.t.TemporalAttemptExecution(get):131 - Stopping cancellation check scheduling...
```

</details>

## Steps to Reproduce
1. Create new source
2. Choose Intercom
3. Put in your access token and test

## Are you willing to submit a PR?
Already done #11176

**Comment by mrhallak** [comment]
@lazebnyi I'm not sure if we can close this one but the PR has been merged

**Comment by lazebnyi** [comment]
Fixed in #11176
